lib/detail_page.dart

```diff
@@ -39,15 +39,16 @@ class _DetailPageState extends State<DetailPage> {
   @override
   Widget build(BuildContext context) {
     final deviceModel = context.watch<DeviceModel>();
+    final navigator = Navigator.of(context);
     return ClipRect(
       child: Theme(
         data: Theme.of(context).copyWith(
           pageTransitionsTheme: YaruPageTransitionsTheme.horizontal,
         ),
         child: Navigator(
```

Comment:
I think it might help simplify the structure a bit and also make it easier to change the release page, if we want to introduce separate upgrade buttons for each release and move away from the selectable list.